automatic/flashplayeractivex/update.ps1

```diff
@@ -0,0 +1,36 @@
+
+import-module au
+
+ $releases = "http://get.adobe.com/en/flashplayer/" # URL to for GetLatest
+
+function global:au_SearchReplace {
+   @{
+        ".\tools\chocolateyInstall.ps1" = @{
+            "(^[$]version\s*=\s*)('.*')"= "`$1'$($Latest.Version)'"
+            "(^[$]majorVersion\s*=\s*)('.*')"= "`$1'$($Latest.majorVersion)'"
+            "(^[$]packageName\s*=\s*)('.*')"= "`$1'$($Latest.PackageName)'"
+            "(^[$]url\s*=\s*)('.*')" = "`$1'$($Latest.URL32)'"
+            "(^[$]checksum\s*=\s*)('.*')" = "`$1'$($Latest.Checksum32)'"
+            "(^[$]checksumType\s*=\s*)('.*')" = "`$1'$($Latest.ChecksumType32)'"
+        }
+    }
+}
+
+function global:au_GetLatest {
+		
+	$HTML = Invoke-WebRequest -Uri $releases
+	$try = ($HTML.ParsedHtml.getElementsByTagName(‘p’) | Where{ $_.className -eq ‘NoBottomMargin’ } ).innerText
```

Comment:
left/right quotation marks used, needs to be normal single quotation mark otherwise it will blow up.